InstaDogFactory.swift

```diff
@@ -49,15 +49,21 @@ class InstaDogFactory {
                         instaDogObject["dog_id"] as? String,
                     let instaDogInstagramURLString: String =
                         instaDogObject["instagram"] as? String,
+                    let instaDogFollowers: String =
```

Comment:
Make sure you check that all your API keys are spelled/written exactly as they appear from the JSON you are provided or else your guard statement will go to the else condition. Even one misspelling and the whole thing is wrong. Do you understand what I mean? You're not able to grab the dogs because this parsing is slightly incorrect.